cmds/wifi/wifi.go

```diff
@@ -53,6 +57,19 @@ func init() {
 	}
 }
 
+func scanWifi() {
+	o, err := exec.Command("iwlist", *iface, "scanning").CombinedOutput()
```

Comment:
*iface

Never supplied as an input from the web interface, right?

Reply:
Nope, only from the cmd